lib/internal/modules/esm/loader.js

```diff
@@ -185,6 +203,39 @@ class Loader {
     if (transformSource !== undefined) {
       this._transformSource = FunctionPrototypeBind(transformSource, null);
     }
+    if (getGlobalPreloadCode !== undefined) {
+      this._getGlobalPreloadCode =
+        FunctionPrototypeBind(getGlobalPreloadCode, null);
+    }
+  }
+
+  runGlobalPreloadCode(context = null) {
```

Comment:
Is context ever used?

Reply:
Not right now. We could remove this code path because we don't support running the preload for anything but the primary context.

Reply:
Done.